src/OrchardCore.Modules/OrchardCore.Media/GraphQL/MediaFieldQueryObjectType.cs

```diff
@@ -23,11 +23,12 @@ public MediaFieldQueryObjectType()
                     {
                         return Array.Empty<string>();
                     }
+
                     return x.Page(x.Source.Paths);
                 });
 
             Field<ListGraphType<StringGraphType>, IEnumerable<string>>("fileNames")
-                .Description("the media fileNames")
+                .Description("the media file names")
```

Comment:
Isn't this now redundant and can be deprecated? (But kept for now for backward compatibility.)

Reply:
I second that. It would be great if we can remove the old redundant fields. It's a breaking change, but we already have lot's of them, so why not take this as well...

Reply:
I also think it's a good time to remove them in 2.0.
@Skrypt what do you think? Or remove them in a new PR?